packages/mosip-api/src/routes/event-review.ts

```diff
@@ -40,19 +29,26 @@ export const reviewEventHandler = async (
       .send({ error: "Token is missing in Authorization header" });
   }
 
-  // Initial test of the verification, we will verify only other informants than mother and father
-  if (stubValidNIDs.includes(informantNationalID)) {
+  const {
+    response: { authStatus },
+  } = await verifyNid({
+    nid: informantNationalID,
+    // @TODO: Figure out the actual DOB
```

Comment:
I'll amend this later when a discussion is being had on the Slack channel. We can still merge this IMO for me to start fixing conflicts on the mediator PR.